### PR TITLE
Initial Setup

### PR DESCRIPTION
**Pull request by mustofa-binary**
- Gatsby setup
- linter
- readme
- translation setup
- in context translation setup
- sitemap and traffic setup
- gtm setup
- SEO setup
- hreflang setup
- Setup theme and responsive
- Create basic components

### CODE REVIEW

File: crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+project_id: '420534'
+api_token_env: 'DERIV_COM_CROWDIN_API_KEY'
```
Comment (oskar-binary) [inline crowdin.yml]: Should make this specific to `DERIV_CRYPTO`

File: src/themes/theme.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { size } from './device'
+
+export const theme = {
+    font: {
```
Comment (oskar-binary) [inline src/themes/theme.js]: should be used in typography as well?

File: src/common/constants.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export const app_name = 'derivcrypto'
+export const app_url = 'derivcrypto.com'
+export const deriv_app_url = 'app.deriv.com'
```
Comment (negarn) [inline src/common/constants.js:4]: how about `app.derivcrypto.com`?

File: src/pages/404/index.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Layout, SEO } from 'components/page-elements'
+import { Text } from 'components/elements'
+import { localize, WithIntl } from 'components/localization'
+
+const PageNotFound = () => {
+    return (
+        <Layout>
+            <SEO
+                title={localize('404 - Page not found | Deriv')}
```
Comment (negarn) [inline src/pages/404/index.js]: should this be deriv crypto?

File: src/websocket/config.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
+import { isBrowser } from '../common/utility'
+
+const domain_config = {
+    production: {
+        hostname: 'derivcrypto.com',
+        app_id: 1407,
+    },
+    staging: {
+        hostname: 'dev.derivcrypto.com',
```
Comment (negarn) [inline src/websocket/config.js]: is this correct?

Comment (mustofa-binary) [reply src/websocket/config.js]: has to be staging.derivcrypto.com

**Review by oskar-binary — DISMISSED** [review]

File: src/components/page-elements/seo/seo.js [hunk 6]
```diff
@@ -50,7 +51,7 @@ export const SEO = ({ description, title, no_index }) => {
             htmlAttributes={{
                 lang,
             }}
-            title={title}
+            title={title + ' - Derivcrypto'}
```
Comment (negarn) [inline src/components/page-elements/seo/seo.js]: we can't do it this way?
```suggestion
            title={`${title} - Derivcrypto`}
```

**Review by negarn — APPROVED** [review]

**Pull request merged**